src/containers/StructurePage/folderComponents/menuOptions/ChangeSubjectName.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectT, tType } from '@ndla/i18n';
+import { Pencil } from '@ndla/icons/action';
+import RoundIcon from '../../../../components/RoundIcon';
+import MenuItemEditField from './MenuItemEditField';
+import { updateSubject } from '../../../../modules/taxonomy';
+import MenuItemButton from './MenuItemButton';
+
+const ChangeSubjectName = ({
+  toggleEditMode,
+  onClose,
+  editMode,
+  name,
+  id,
+  contentUri,
+  getAllSubjects,
+  refreshTopics,
+  t,
+}: Props & tType) => {
+  const onChangeSubjectName = async (name: String) => {
```

Comment:
```suggestion
  const onChangeSubjectName = async (name: string) => {
```

Skal det være liten s og ikke stor s her, for å skille på TypeScript typen string og JavaScript klassen String? @N4G1 forklarte dette en gang, he knows.

Reply:
Det er sant. Det tenkte ikkje eg på.

Reply:
Det stemmer @elenaand, bra catch!